src/commcare_cloud/ansible/letsencrypt_cert.yml

```diff
@@ -108,4 +110,13 @@
         hour: "0/12"
         user: root
         cron_file: certbot
-        state: absent
+
+    - name: reload nginx every 15 days
```

Comment:
Does letsencrypt already automatically update the cert every 15 days?

Reply:
It only updates in it 30 days.